common/src/main/java/com/wynntils/core/features/FeatureRegistry.java

```diff
@@ -138,16 +137,15 @@ public static void init() {
         registerFeature(new ConnectionProgressFeature());
         registerFeature(new PacketDebuggerFeature());
 
-        // internal
+        // stateless
```

Comment:
"stateless" in this terminology is supposed to mean "non-enableable"? 

I'm not saying the latter is a better word, but "stateless" is very general, and makes it more sound like there is no fields, which is patently false. So we might need to work on the  name a bit.

"always-on"? "permanent"?